lib/token-assert.js

```diff
@@ -143,7 +143,7 @@ TokenAssert.prototype.differentLine = function(options) {
 TokenAssert.prototype.linesBetween = function(options) {
     var token = options.token;
     var nextToken = options.nextToken;
-    var whitespaceBefore = ' ';
+    var whitespaceBefore = '';
```

Comment:
this should remain a `' '` in the case where the two tokens joining would result in nonsense, e.g.:

```
typeof
a === 'function';
```
->
```
typeofa === 'function';
```